.syncpackrc

```diff
@@ -112,7 +112,10 @@
 		{
 			"dependencies": [
 				"@wordpress/block**",
-				"@wordpress/viewport"
+				"@wordpress/viewport",
+				"@wordpress/interface",
+				"@wordpress/router",
+				"@wordpress/edit-site"
```

Comment:
These are all newly added packages. We pin `@wordpress/*` packages version in wp 6.0. However, we need to use different versions for these packages so I change this setting to ignore them.